platforms/android/library/src/main/java/io/element/android/wysiwyg/utils/RustCleanerTask.kt

```diff
@@ -1,13 +1,16 @@
 package io.element.android.wysiwyg.utils
 
+import io.element.android.wysiwyg.BuildConfig
 import timber.log.Timber
 import uniffi.wysiwyg_composer.Disposable
 
 internal class RustCleanerTask(
     private val disposable: Disposable,
 ) : Runnable {
     override fun run() {
-        Timber.d("Cleaning up disposable: $disposable")
+        if (BuildConfig.DEBUG) {
```

Comment:
I added this because the debug logs added here were being printed by the EXA app in debug mode. I could also remove the log completely.